src/scss/styles.scss

```diff
@@ -1,20 +1,51 @@
 @import "bourbon";
 @import "neat";
 
+@import url(http://fonts.googleapis.com/css?family=Raleway:400,600,700,300);
+@import url(http://fonts.googleapis.com/css?family=Roboto:400,600,700,300);
+
+
+// Base
 $base-border-radius: 3px !default;
 $base-line-height: 1.5em !default;
 $base-spacing: 1.5em !default;
-$action-color: #477DCA !default;
-$dark-gray: #333 !default;
-$light-gray: #DDD !default;
+$base-heading-top-spacing: 1em !default;
+
+// Media Queries
 $medium-screen: em(640) !default;
-$base-font-color: $dark-gray !default;
+
+// Color Palette
+// Generated From https://www.materialpalette.com/grey/blue
+$primary-color-dark:   #616161 !default;
+$primary-color:        #9E9E9E !default;
+$primary-color-light:  #F5F5F5 !default;
+$primary-color-text:   #212121 !default;
+$accent-color:         #448AFF !default;
+$primary-text-color:   #212121 !default;
+$secondary-text-color: #727272 !default;
+$divider-color:        #B6B6B6 !default;
+
+$action-color: $accent-color;
+$dark-gray: $primary-text-color;
+$light-gray: $primary-color-light;
+$base-font-color: $primary-text-color;
+
+// Type
 $serif: 'Lusitana', serif;
 $serif-2: 'Merriweather', serif;
+$serif-3: 'Raleway', serif;
+$serif-4: 'Roboto', serif;
 $sans-serif: 'Merriweather Sans', sans-serif;
 
+// Brand Styles
+$brand-primary-type: $serif-2;
+$brand-primary-type-sans-serif: $sans-serif;
+$brand-type-headings: $serif-3;
+
 
-//* General Site Styles
+/* ==========================================================================

```

Comment:
Not sure what your preference for comment styling is but I use the Sublime package [CSS Comments](https://github.com/brenopolanski/css-comments-sublime-snippets).